build/ci-scripts/api-testing.config.json

```diff
@@ -0,0 +1,12 @@
+{
+	"base_uri": "http://localhost:8484/w/",
+	"main_page": "Main_Page",
+	"root_user": {
+		"name": "admin",
+		"password": "secret-password"
+	},
+	"secret_key": "abcdef",
```

Comment:
Shouldn’t this be “secretkey” according to `MW_WG_SECRET_KEY` in `docker-compose.yml` line 21?

Reply:
yep, done!

Reply:
Seems like the fix got lost in some rebase :/